### ISSUE TITLE
Multiplied by y_true once to much

### ISSUE THREAD

**Issue by gartangh** [aldi-dimara/keras-focal-loss#1]
I believe you multiplied once to much with `y_true` in `categorical_focal_loss()`.
Where you calculate `weight`, `y_true` can be left out.
```python
# Calculate cross entropy
cross_entropy = -y_true*K.log(y_pred)
# Calculate weight that consists of  modulating factor and weighting factor
weight = alpha * y_true * K.pow((1-y_pred), gamma)
# Calculate focal loss
loss = weight * cross_entropy
```
